__tests__/app.test.js

```diff
@@ -34,8 +36,53 @@ describe("GET /api/topics", () => {
       });
   });
 });
-describe("/api/invalid-input", () => {
+describe("GET /api/invalid-input", () => {
   test("GET:404 responds with a 404 when given an invalid path", () => {
     return request(app).get("/api/invalid-input").expect(404);
   });
 });
+
+describe("GET /api", () => {
+  test("responds with an object describing all the available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.endpoints).toEqual(endpoints);
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET: 200 responds with an array of object with all the info of the correct article_id ", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual([
+          {
+            article_id: 3,
```

Comment:
think about using a matcher here eg expect.objectContaining or .toMatchObject - will allow for extension later